include/ibamr/BrinkmanAdvDiffSemiImplicitHierarchyIntegrator.h

```diff
@@ -150,6 +150,23 @@ class BrinkmanAdvDiffSemiImplicitHierarchyIntegrator : public AdvDiffSemiImplici
         return d_brinkman_penalization;
     } // getBrinkmanPenalization
 
+    /*!
+     * \brief Set the boolean to be TRUE for the time independent Brinkman problems.
+     * Default option is false.
```

Comment:
\brief Indicate if are solving a time independent problem for the Brinkman variable Q.
     * Default option is false, which means Q is assumed to vary with time.

Reply:
We still need to make this correction.